src/App.tsx

```diff
@@ -5,7 +5,18 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const addMovie = (newMovie: Movie) => {
+    if (movies.find(movie => movie.imdbId === newMovie.imdbId)) {
```

Comment:
```suggestion
    if (movies.some(movie => movie.imdbId === newMovie.imdbId)) {
```
`find` method returns you whole object, if you need here only boolean it will be better to use `some`